### ISSUE TITLE
potential issue with `write_summary_tsv`

### ISSUE THREAD

**Issue by fbergmann** [PEtab-dev/petab_select#37]
in line 272 in ui.py we have 

```python 
    # FIXME remove once MostDistantCandidateSpace exists...
    method = candidate_space.method
    if (
        candidate_space.governing_method == Method.FAMOS
        and candidate_space.predecessor_model.predecessor_model_hash is None
    ):
```

however, the `predecessor_model` can be either a Model, or str. In the case of str this code will fail with an AttributeError.

**Comment by dilpath** [comment]
Hi Frank, thanks! Should be fixed in #38 and available here and via PyPI.

**Comment by fbergmann** [comment]
thanks that fixes this issue.
